### ISSUE TITLE
Dead pull/push task is never rescheduled

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#1416]
When the push_all or pull_all task fails/hangs/dies it is not rescheduled. This means an admin has to manually reschedule it before it starts running again.

Perhaps the scheduling mechanism should detect dead/hanging tasks and reschedule them according to their frequency setting.

**Comment by iglocska** [comment]
Yup we definitely will need something for this. It's annoying that we have no way of letting the tasks self-heal after issues.
